src/transactions/TransactionUtils.cpp

```diff
@@ -639,6 +639,13 @@ getMinBalance(LedgerHeader const& header, AccountEntry const& acc)
                          numSponsored);
 }
 
+int64_t
+getMinBalance(Application& app, AccountEntry const& acc)
```

Comment:
Nitpick: I think this should live in a testing file, because this is not actually safe to use in the transaction subsystem (creating a child `LedgerTxn` when one already exists will throw, and a child `LedgerTxn` always exists within the transaction subsystem).

Reply:
Moved